index.html

```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h1>My recent works</h1>
```

Comment:
- [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.